python-package/rgf/rgf_model.py

```diff
@@ -450,9 +504,20 @@ def __init__(self,
         '{%n_classes_property%}': ''
     }
     __doc__ = rgf_estimator_docstring_template
-    for _template, _value in _regressor_specific_values.items():
+    for _template, _value in _regressor_init_specific_values.items():
```

Comment:
Why starts from "_"?

Reply:
Due to prevent IDE autocomplete with this variable and not list it as a class member.
Refer to https://github.com/RGF-team/rgf/pull/195#pullrequestreview-135827019.

Reply:
No problem! :-)